CMakeLists.txt

```diff
@@ -139,6 +139,8 @@ cmake_dependent_option(
     "USE_GLOO" OFF)
 option(USE_TBB "Use TBB" OFF)
 option(USE_EIGEN_THREADPOOL, "Use Eigen thread pool" OFF)
+option(USE_MKL_SEQ "Use sequential MKL" OFF)
```

Comment:
Instead of this maybe make it flag you pass a string to? Makes it easier to keep an overview too and warn on a wrong setting.